hyperledger/grid-docs
# Fix links in doc and website

Fix broken links and missing redirects, update links with recent changes, add links to new documents, and change link format to use the release-neutral Liquid variable when possible.

- Add redirect files in all subdirectories to avoid broken links (from old bookmarks) and to prevent hand-crafted bookmarks from seeing a directory listing.

- Website landing page: Fix links on current landing page (update the link to Hyperledger's Project Lifecycle page; remove the link to Hyperledger Grid in the news).
      - **NOTE:** A separate PR proposes to delete this list of links. This PR fixes the existing links until that PR is merged or replaced.

- Grid Roadmap: 
     - Update CLI links to go to the new man pages
     - Add new doc links to Grid Roadmap: Location specification, Pike (organization) how-to procedure, Product overview and how-to procedure

- Community/Best Practices/Feature documentation:
     - Use release-neutral links (change doc HREF links to use the Liquid variable site.data.general.latest_version)
     - Fix links and content in the CLI Command Reference and Man Pages section (add CLI Usage Statement section; under CLI Command Reference, note man pages are now automatically included in the CLI ref)

- Delete a duplicate "Running Grid on Splinter" link from the doc's left-nav TOC.

**Test locally:**

- Run this in the root directory: `just run`

- Wait until you see `grid-docs-jekyll | Server running... press ctrl-c to stop.`

- Point your browser to `0.0.0.0:8080`
